checkstyle/suppressions.xml

```diff
@@ -25,6 +27,8 @@
               files="Fetcher.java"/>
     <suppress checks="ParameterNumber"
               files="ConfigDef.java"/>
+    <suppress checks="ParameterNumber"
+              files="EosLogEntry.java"/>
```

Comment:
We'd need to increase a bit more to capture this one too. Increasing to 12 dropped two others.